spec/features/goal_tool_spec.rb

```diff
@@ -15,7 +15,7 @@
   end
 
   scenario "Displays due date" do
-    expect(page).to have_content "Due: #{Date.today.to_s(:date)}"
+    expect(page).to have_content "Due: #{Date.today.strftime("%b %d %Y")}"
```

Comment:
Should you use the format defined here: config/initializers/datetime_formats.rb, instead of a static string?